homeassistant/components/simplisafe/__init__.py

```diff
@@ -353,24 +353,18 @@ async def set_system_properties(call: ServiceCall) -> None:
     ):
         async_register_admin_service(hass, DOMAIN, service, method, schema=schema)
 
-    current_options = {**entry.options}
```

Comment:
I think you will still have to make a copy because I'm pretty sure that [this code](https://github.com/home-assistant/core/blob/dev/homeassistant/config_entries.py#L1058) will update the `entry.options` since its the same object.